frontend/uddhar-frontend/src/organization/pages/OngoingDisaster.jsx

```
@@ -0,0 +1,52 @@
+          <>
+            {ongoingDisaster.map((disaster) => (
+              <div
+                key={`${disaster.title}-${disaster.description}`}
+                className="bg-white shadow-lg rounded-lg p-6 m-10 relative overflow-hidden bg-cover bg-center text-white transform hover:scale-105 transition-transform duration-300"
+                style={{
+                  backgroundImage: `url(${disaster.image})`,
+                  width: "350px",
+                  height: "260px",
+                }}
+                onClick={handleNext}
+              >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)